charts/prometheus-snmp-exporter/values.yaml

```
@@ -185,8 +185,20 @@ serviceMonitor:
 
   # Metric relabeling is applied to samples as the last step before ingestion.
   # Set if defined unless overriden by params.additionalMetricsRelabels.
+  # This sets fixed relabel configs with action 'replace'.
   additionalMetricsRelabels: {}
```

Thanks for updating the comment above to be clearer. Please could you also add these comments beneath `additionalMetricsRelabels` to provide an example of how can be used
```suggestion
  additionalMetricsRelabels: {}
    # targetLabel1: replacementValue1
    # targetLabel2: replacementValue2
```